sound/hybrid_font.h

```diff
@@ -799,8 +803,44 @@ class HybridFont : public SaberBase, public Looper {
 	SaberBase::DoEffect(EFFECT_POSTOFF, saved_location_);
       }
     }
+    // Wait for retraction sounds to finish, then play idle.wav
+#ifdef ENABLE_IDLE_SOUND
+    if (idling_) {
+      if (GetWavPlayerPlaying(&SFX_in) || GetWavPlayerPlaying(&SFX_pstoff)) {
+        return;
+      } else {
+        idling_ = false;
+        StartIdleSound();
+      }
+    }
+#endif
+  }
+  
+  void StopIdleSound() {
+  #ifdef ENABLE_IDLE_SOUND
+    idling_ = false;
+    if (idle_player_ && idle_player_->isPlaying()) {
```

Comment:
Should we maybe play some other effect here?
(a sort of idle "out" sound?)

Reply:
My feeling is it's a single event sound, just a background ambience, sfx, whooshing, humming, beeping, whatever. No actual "HERE'S THE IDLE SOUND!" announcement. Although if you want to add something here we could, or leave it for now and add it later if the "need" arises?

Reply:
If the sounds were built similar to lockups, I could see a bgnidle.wav maybe being an option, yeah.